plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/util/SearchUtils.java

```diff
@@ -192,7 +208,7 @@ public final String getESResultJson(final String esQueryJson, final String url)
     private String handleHttpException(final Exception e) {
         KBNotification.getInstance().error(e);
         e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)